### ISSUE TITLE
connect seems to be ignoring password

### ISSUE THREAD

**Issue by ccurvey** [PyMySQL/PyMySQL#417]
Given the following script:

    import MySQLdb
    import pymysql

    old_conn = MySQLdb.connect(host=HOST, user=USER, passwd=PASSWD, port=PORT,
                           db=DATABASE)

    new_conn = pymysql.connect(host=HOST, user=USER, passwd=PASSWD, port=PORT,
                           db=DATABASE)

The second connect request raises an error:

    pymysql.err.OperationalError: (1045, u"Access denied for user 'obfuscated'@'obfuscated' (using password: NO)")

It looks like the password is being ignored.  If I connect to that server with the mysql command line client, I get:

    Server version: 5.1.49-log yes

**Comment by fabiolab** [comment]
Same for me !
```
import pymysql.cursors
connection = pymysql.connect(host="xxxx",
                            user="rii",
                            password="rii",
                            db="RCA",
                            charset='utf8',
                            cursorclass=pymysql.cursors.DictCursor)

OperationalError: (1045, "Access denied for user 'rii'@'xxx' (using password: NO)")
```

It works when downgrading to 0.6.7 version of PyMySQL.
Thanks for the all the job about this lib !

**Comment by methane** [comment]
@fabiolab Which version of MySQL do you use?

**Comment by methane** [comment]
This bug may be introduced by auth plugin support.
But I can't get MySQL 5.1 easily...

**Comment by ccurvey** [comment]
I have a VM set up with MySQL 5.1.  What is the best way to get credentials to you?

**Comment by methane** [comment]
I can use Docker on local machine.

Though Travis supports docker, current `.travis.yml` is too complicated and not easy to use docker.

**Comment by methane** [comment]
I've released 0.7.1 on PyPI.
Please test it.

**Comment by fabiolab** [comment]
Your fix is ok ! Great (and so quick) job, thanks !

(Tested on MySql 5.0.77).

**Comment by ccurvey** [comment]
confirmed fix with MySQL 5.1.49. thanks for the quick turnaround!
